docs/chapters/1-installation.md

```diff
@@ -56,42 +57,44 @@ iCloud3 is listed on the default HACS Repositories/Integrations page and can be
 !> iCloud3 logs information to the HA log file and to an internal table that can be viewed using the iCloud3 Event Log Lovelace Custom Card. Information about this custom card, and installation instructions are in the Support Programs chapter.
 
 ### About the iCloud3 Event Log Custom Card
-As iCloud3 runs, various entries are written to the HA log file that show device information, how it is tracked, operational errors, startup information and other items that may help determine what is going on if there is a problem and to monitor when the device's information is determined and updated. A lot of this information is also written to the `iCloud3 Event Log` which can be viewed using the `iCloud3 Event Log Lovelace Card`. 
+As iCloud3 runs, various entries are written to the HA log file that show device information, how it is tracked, operational errors, startup information and other items that may help determine what is going on if there is a problem and to monitor when the device's information is determined and updated. A lot of this information is also written to the `iCloud3 Event Log` which can be viewed using the `iCloud3 Event Log Lovelace Card`.
 
 Below are 3 screens. The one on the left shows iCloud3 starting up, the middle one shows arriving Home and the one on the right shows entering the 'Whse' zone.
 
 ![event_log](../images/event_log_initializing.jpg)
 
 ---
 ### Installing the iCloud3 Event Log Custom Card
-Custom Lovelace cards are typically stored in the `/www/custom_cards` directory. Do the following:
+Custom Lovelace cards are typically stored in the `config/www/custom_cards` directory. Do the following:
 
-1. Create the `/www/custom_cards` directory if it does not exist. 
+1. Create the `config/www/custom_cards` directory if it does not exist.
 
-2. Copy the `icloud3-event-log-card.js` into the `/www/custom-cards` directory. If you are installing iCloud3 using HACS, the file is in the `custom_components/icloud3` directory. If you are already using custom cards and they are in a different directory,  copy the `icloud3-event-log-card.js` file into the one you are using.
+2. Copy `icloud3-event-log-card.js` into the `config/www/custom-cards` directory. If you are installing iCloud3 using HACS, the source file is in the `config/custom_components/icloud3` directory. If you are already using custom cards and they are in a different directory, copy `icloud3-event-log-card.js` into the one you are using.
 
-3. Open the `ui-lovelace.yaml` file and add the following lines to the beginning of the file. Again, change the directory name if you are using a different location.
+3. Open the file `configuration.yaml`, and add similar lines below to the lovelace section to add the new resource.
```

Comment:
Resource file are no longer allowed to be in the ui-lovelace.yaml file. :(

https://community.home-assistant.io/t/warning-resources-need-to-be-specified-in-your-configuration-yaml/191655